src/models/forecasts.jl

```diff
@@ -99,7 +99,7 @@ function SystemForecasts(data::NamedTuple)
     return SystemForecasts(_Forecasts(), initial_time, resolution, horizon, interval)
 end
 
-function Base.show(io::IO, forecasts::SystemForecasts)
+function Base.summary(io::IO, forecasts::SystemForecasts)
```

Comment:
On second thought, I think summary should be used here, which is what it was originally.